{"ip_str": "104.238.182.217", "errors": {"VPN_error": "Timeout exceeded.\n<pexpect.pty_spawn.spawn object at 0x7f99e240f370>\ncommand: /usr/sbin/openvpn\nargs: ['/usr/sbin/openvpn', '/home/NetPlatform/configurations/bulletVPN-US-Los_Angeles_1-lax01-tcp.ovpn']\nbuffer (last 100 chars): b'),init_instance] received, process restarting\\r\\nTue Nov  1 08:54:55 2022 Restart pause, 5 second(s)\\r\\n'\nbefore (last 100 chars): b'),init_instance] received, process restarting\\r\\nTue Nov  1 08:54:55 2022 Restart pause, 5 second(s)\\r\\n'\nafter: <class 'pexpect.exceptions.TIMEOUT'>\nmatch: None\nmatch_index: None\nexitstatus: None\nflag_eof: False\npid: 9\nchild_fd: 5\nclosed: False\ntimeout: 30\ndelimiter: <class 'pexpect.exceptions.EOF'>\nlogfile: None\nlogfile_read: None\nlogfile_send: None\nmaxread: 2000\nignorecase: False\nsearchwindowsize: None\ndelaybeforesend: 0.05\ndelayafterclose: 0.1\ndelayafterterminate: 0.1\nsearcher: searcher_re:\n    0: re.compile(b'Initialization Sequence Completed')"}}